test/negative_tests/test_reconcile_logs.py

```diff
@@ -0,0 +1,17 @@
+import pytest
```

Comment:
What is a sense to place tests, helpers and static file to a separate directory? We already have a directory for unit tests - `unittest` and I think we can place your pytest tests there too. If you want to run negative tests separately we can use [pytest marks](https://docs.pytest.org/en/6.2.x/mark.html) for this.

Reply:
If we're about my initial idea, those are not unit tests: here we want to run test-run with a test, which is known to fail, and check that it fails as expected: look at the error message, verify that there are no orphan tarantool instances after test-run termination.

Reply:
xfail is the different thing in my understanding: it is when a test should pass, but the code is not ready at the moment. It is the temporary mark, when we expect that the code will be fixed in a future.

My idea is different: add tests that will verify test-run behaviour, when a test under test-run fails. Here we have another success critetia: not just whether a test-run's test passed. But everything else is the same: correct test-run behaviour is not 'expected fail' — it is success.

Reply:
I agree with @Totktonada. `xfail `mark is used in another way. A test can be failed in different situations, for example, unexpected exception, negative testing is a type of testing when we know the behavior of our product.

Reply:
I'll close the discussion: it seems, there is nothing more to discuss. Sergey, if you have something else to say here, please, reopen.